packages/ui/.eslintrc

```diff
@@ -15,6 +15,20 @@
             "never"
         ],
         "lodash/import-scope": "error",
+        "no-restricted-imports": ["error", {
+            "paths": [
+                {
+                    "name": "@gravity-ui/uikit",
+                    "importNames": ["Dialog"],
```

Comment:
```ts
"name": "@gravity-ui/dialog-fields",
"importNames": ["DFDialog"],
```
And `// eslint-disable-next-line no-restricted-imports` should be also present for the rule